mpact/tasks.py

```diff
@@ -40,18 +42,15 @@ async def send_msg_task(receiver_id, message):
         message_data[SENDER_NAME] = current_bot.first_name
         message_data[ROOM_ID] = int(receiver_id)
         message_data[MESSAGE] = message
-        try:
-            group_chat = Chat.objects.get(id=receiver_id)
-            message_data[FROM_GROUP] = True
 
-        except Chat.DoesNotExist:
-            individual_chat = Individual.objects.get(id=receiver_id)
-            message_data[FROM_GROUP] = False
+        chat_or_group = get_chat_by_telegram_id(receiver_id)
+        group_mode = isinstance(chat_or_group, Chat)
+        message_data[FROM_GROUP] = group_mode
```

Comment:
I think because of the previously selected naming convention, the variable name `chat_or_group` is confusing. As identified in the previous commit, `Chat` is a group and `Individual` is a 1:1. Line 47 makes it clear, but looks like a potential bug without tracing down `Chat`.

Should we rename the `Chat` and `Individual` models to make code like this (and everywhere else) more clear?

Reply:
(If it helps naming, the API has a [chat](https://core.telegram.org/tdlib/docs/classtd_1_1td__api_1_1chat.html) class, which can be a private chat (which I assume means a 1:1), a [basicGroup](https://core.telegram.org/tdlib/docs/classtd_1_1td__api_1_1basic_group.html), a [supergroup](https://core.telegram.org/tdlib/docs/classtd_1_1td__api_1_1supergroup.html) or a [secretChat](https://core.telegram.org/tdlib/docs/classtd_1_1td__api_1_1secret_chat.html). ... :thinking: I'm not sure that does help, actually.)

Reply:
Good call on the confusing variable name, and on renaming the model. Changing the model names at this stage shouldn't be hard. I'm also interested in exploring whether we can use a model inheritance pattern similar to how the API docs do (thanks for the pointer). I'll propose some new names in a follow up. Think there is a lot of model cleanup to be done, though unclear how much will happen now.